src/hotspot/share/cds/filemap.hpp

```diff
@@ -554,6 +555,8 @@ class FileMapInfo : public CHeapObj<mtInternal> {
                     GrowableArray<const char*>* rp_array,
                     unsigned int dumptime_prefix_len,
                     unsigned int runtime_prefix_len) NOT_CDS_RETURN_(false);
+  bool  is_jar_suffix(const char* filename);
```

Comment:
Suggestion: has_jar_suffix

Reply:
Fixed. Also moved the function to ClassLoaderExt.cpp.